src/foolscap/_version.py

```diff
@@ -86,20 +86,20 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
             if e.errno == errno.ENOENT:
                 continue
             if verbose:
-                print("unable to run %s" % dispcmd)
+                print(("unable to run %s" % dispcmd))
                 print(e)
```

Comment:
why double parens here? (and similarly elsewhere)

Reply:
Looks like its created by the 2to3 script. Weird.